packages/history-utility/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "valtio-history",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
you dont need to adjust this.. its automated when releasing

Reply:
Ok, rolled back